geomesa-tools/README.md

```diff
@@ -931,3 +931,174 @@ Prints out the version, git branch, and commit ID that the tools were built with
+           Number of partitions for the Kafka topic
+        --replication
+           Replication factor for Kafka topic
+      * -s, --spec
+           SimpleFeatureType specification as a GeoTools spec string, SFT config, or
+           file with either
+      * -p, --zkpath
+           Zookeeper path where feature schemas are saved
+      * -z, --zookeepers
+           Zookeepers (host[:port], comma separated)
+
+#### Example command:
+    
+    $ geomesa-kafka create -f testfeature \
+      -z zoo1,zoo2,zoo3 \
+      -b broker1:9092,broker2:9092 \
+      -s fid:String:index=true,dtg:Date,geom:Point:srid=4326 \
+      -p /geomesa/ds/kafka
+
+### describe
+Display details about the attributes of a specified feature type.
+
+#### Usage:
+
+    $ geomesa-kafka help describe
+    Describe the attributes of a given feature in GeoMesa
+    Usage: describe [options]
+      Options:
+      * -b, --brokers
+           Brokers (host:port, comma separated)
+      * -f, --feature-name
+           Simple Feature Type name on which to operate
+      * -p, --zkpath
+           Zookeeper path where feature schemas are saved
+      * -z, --zookeepers
+           Zookeepers (host[:port], comma separated)
+
+#### Example command:
+
+    $ geomesa-kafka describe -f testfeature -z zoo1,zoo2,zoo3 -b broker1:9092,broker2:9092 -p /geomesa/ds/kafka
+
+### list
+List all known feature types in Kafka. If no zkpath parameter is specified, the list command will search all of zookeeper for potential feature types.
+
+#### Usage:
+
+    $ geomesa-kafka help list
+    List GeoMesa features for a given zkPath
+    Usage: list [options]
+      Options:
+      * -b, --brokers
+           Brokers (host:port, comma separated)
+        -p, --zkpath
+           Zookeeper path where feature schemas are saved
+      * -z, --zookeepers
+           Zookeepers (host[:port], comma separated)
+
+
+#### Example command:
+
+    $ geomesa-kafka list -z zoo1,zoo2,zoo3 -b broker1:9092,broker2:9092
+
+### listen
+Logs out the messages written to a topic corresponding to the passed in feature type.
+
+#### Usage:
+
+    $ geomesa-kafka help listen
+    Listen to a GeoMesa Kafka topic
+    Usage: listen [options]
+      Options:
+      * -b, --brokers
+           Brokers (host:port, comma separated)
+      * -f, --feature-name
+           Simple Feature Type name on which to operate
+        --from-beginning
+           Consume from the beginning or end of the topic
+           Default: false
+      * -p, --zkpath
+           Zookeeper path where feature schemas are saved
+      * -z, --zookeepers
+           Zookeepers (host[:port], comma separated)
+
+#### Example command:
+
+    $ geomesa-kafka listen -f testfeature \
+      -z zoo1,zoo2,zoo3 \
+      -b broker1:9092,broker2:9092 \
+      -p /geomesa/ds/kafka \
+      --from-beginning
+
+### removeschema
+Used to remove a feature type `SimpleFeatureType` in a GeoMesa catalog. This will also delete any feature of that type in the data store.
```

Comment:
Wondering if we should be more explicit about the fact we delete the topic...user may not want to delete it. not sure. or at least doc it.